build.gradle

```diff
@@ -0,0 +1,125 @@
+buildscript {
+    dependencies {
+        classpath files('gradle/gradle-witness.jar')
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'witness'
+apply plugin: 'maven-publish'
+
+compileJava {
+    sourceCompatibility = 1.7
+    targetCompatibility = 1.7
+}
+
+version = "37"
+
+repositories {
+    mavenLocal()
+    maven { url "http://4thline.org/m2" }
+    maven { url 'https://mvn.freenetproject.org' }
+    jcenter()
+}
+
+configurations {
+    extraLibs
+}
+
+dependencies {
+    compile group: 'org.freenetproject', name: 'fred', version: 'build+'
+}
+
+dependencyVerification {
+    verify = [
+        'org.freenetproject:fred:a181e584a055211ebbe69657a89f8e1452a5eb72af441839208223dc6515cbce',
```

Comment:
Hmm, so you're pinning it to a specific version of fred?
IMHO that will break way too often

Reply:
Thanks for the time to review this. I'm not sure the pinning thing as the dependency is defined as:

```
dependencies {
    compile group: 'org.freenetproject', name: 'fred', version: 'build+'
}
```

I don't know much about gradle and/or fred version management so I don't know what "build+" actually means.

But for sure the `dependencyVerification` has to have specific hashes for whatever version was active at that build time.

Reply:
build+ means "any version of fred" ... and that's obviously incompatible with that line :)

I'd suggest that you leave all except this line (and yes, it will break when the fred dependencies change... but that's hardly ever happening nowadays).